feedstock/catalog.yaml

```diff
@@ -1,9 +1,14 @@
 # All the information important to cataloging.
-data_store_prefix: "gs://leap-scratch/jbusecke/proto_feedstock/latest/data"
-catalog_store_prefix: "gs://leap-scratch/jbusecke/proto_feedstock/latest/catalog"
-recipes:
-  small:
-    pyramid_store_path: "gs://some-bucket/some-path"
-    extra_metadata:
-      field: "value"
-      another_field: "another_value"
+"ncviewjs:meta_yaml_url": "https://github.com/leap-stc/proto_feedstock/blob/main/feedstock/meta.yaml"
+tags:
+  - my-custom-tag
+  - zarr
+stores:
+  - name: small.zarr
```

Comment:
@andersy005 Should we stick with the same syntax as meta.yaml here? 
```
stores:
    - id: small # needs to match meta.yaml id?
       name: "A cool name to display in the catalog"
       url: "gs://leap-scratch/jbusecke/proto_feedstock/latest/data/small.zarr"
```

Reply:
👍🏽 for conforming to the same syntax as meta.yaml. it will come in handy when organizing/compiling entries in catalog.yaml. we could make `id` and `url` required fields and `name` optional



 as far as the leap-catalog is concerned, we can always check to see if `name` is specified, otherwise fall back to `id`.

Reply:
Perfect. I am just commiting these and testing the recipe execution.